FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_plaintext/using_plaintext.c

```diff
@@ -182,5 +183,15 @@ int32_t Plaintext_FreeRTOS_send( NetworkContext_t * pNetworkContext,
         socketStatus = 0;
     }
 
+    #if ( configUSE_PREEMPTION == 0 )
+    {
+    /* Allow the IP task to instantly send the packets through the ethernet interface.
+     * The call to taskYIELD() fixes a bug in collaborative mode where the MQTT_Connect
+     * function attempts to read CONNACK before CONNECT is sent, resulting in
+     * MQTTNoDataAvailable error. */
```

Comment:
Thank your for your contribution. Please also indent the comment. It would be great if you could run uncrustify 0.67 to properly format your code using the following config:
https://github.com/FreeRTOS/FreeRTOS/blob/master/tools/uncrustify.cfg